applications/visit/run.sh

```diff
@@ -0,0 +1,196 @@
+set -x
```

Comment:
Now that this is the `interactive-template` file to be used across many apps, can we put this in a new folder called `interactive-template` under `applications`?